cmd/aries-agentd/startcmd/start.go

```diff
@@ -168,3 +193,12 @@ func startAgent(server server, host, inboundHost, dbPath string) error {
 
 	return nil
 }
+
+func startWebhookServer(webhookHost string) {
```

Comment:
Should we call this as webhook server or controller server ? @troyronda @llorllale

Reply:
just spoke to @llorllale to confirm. This isn't the controller, just the actual webhook dispatcher. I'm going to be reworking this code soon, perhaps I can give it a clearer name.